File: Python/CCC/ccc11s1.py
n = int (input())
sCount = 0
tCount = 0
for i in range (n):
    s = str (input())
    for j in range(len(s)):
        if (s[j].lower()=='s'):
            sCount += 1
        elif (s[j].lower()=='t'):
            tCount += 1
if (sCount>=tCount):
    print("French")
else:
    print("English")

File: Java/CCC/ccc11s1.java
import java.util.Scanner;


public class ccc11s1 { //english or french

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int sCount = 0;
		int tCount = 0;
		for (int i=0; i<n; i++) {
			String s = sc.nextLine();
			for (int j=0; j<s.length(); j++) {
				if (s.charAt(j)=='s' || s.charAt(j)=='S') {
					sCount++;
				} else if (s.charAt(j)=='t' || s.charAt(j)=='T') {
					tCount++;
				}
			}
		}
		if (sCount>tCount) {
			System.out.println("French");
		} else {
			System.out.println("English");
		}

	}
}